<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionSettings</class>
 <widget class="KittySDK::ISettingsPage" name="ConnectionSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <property name="name" stdset="0">
   <string notr="true">Settings.Connection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="connectionGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="sleepDisconnectCheckBox">
        <property name="text">
         <string>Disconnect when going to sleep</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="wakeUpReconnectCheckBox">
        <property name="text">
         <string>Reconnect on wake up</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="reconnectCheckBox">
        <property name="text">
         <string>Reconnect after loosing connection</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="proxyGroupBox">
     <property name="title">
      <string>Proxy</string>
     </property>
     <layout class="QFormLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="proxyCheckBox">
        <property name="text">
         <string>Use proxy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="serverLabel">
        <property name="text">
         <string>Server and port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="_2">
        <item>
         <widget class="QLineEdit" name="serverEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="portEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>8080</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="authCheckBox">
        <property name="text">
         <string>Server requires authentication:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="usernameEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="passwordEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>83</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets/>
 <resources/>
 <connections>
  <connection>
   <sender>proxyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>serverEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>portEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>authCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>usernameEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>authCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
